test/loader_test/icd_test_match.c

```diff
@@ -26,14 +26,16 @@ int test_icd_match()
     }
 
     if (strcmp(app_log, stub_log)) {
-        printf("ERROR: App log and stub log differ.\n");
+        printf("ERROR: App log and stub log differ.\n\n");
+        printf("APPLOG:\n%s\n\n", app_log);
+        printf("STUBLOG:\n%s\n\n", stub_log);
         error = 1;
         goto End;
     }
 
 End:
     free(app_log);
-    free(stub_log);
+    //free(stub_log);
```

Comment:
Very quickly following `test_icd_get_stub_log` it looks like this *should* be free'd?

Reply:
Yes, I missed commenting on that. I found that freeing the stub log crashed the test on my local box. Since it letting it leak and be it freed 3 statements later on `return 0;` seemed like a viable "hotfix" I went with that. The logs weren't empty, but free triggered an assertion. (MSVC, Debug build)

What should the remedy be?